test/sagemaker_tests/pytorch/training/resources/mnist/mnist.py

```diff
@@ -24,7 +24,6 @@
 import os
 import sys
 
-import cv2 as cv
```

Comment:
is this package the cause of the hanging tests?

Reply:
yes